### PR TITLE
Removes omni gas devices

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR removes the omni gas devices from the game. They aren't used anywhere, aren't accessible ingame, and are quite broken. Heres the UI for one of them.
image
Tell me how the hell you are supposed to configure this. 

These were added 6 years ago by a coder who is long gone from here, and were never updated throughout their lifespan. They are unused code, and just waiting to be stripped out.

## Why It's Good For The Game
Unused broken code bad

## Changelog
:cl: AffectedArc07
del: Removed some old, inaccessible atmos devices. There should be no ingame changes.
/:cl:

### CODE REVIEW

File: code/__DEFINES/pipes.dm [hunk 1]
```diff
@@ -21,8 +21,6 @@
 #define PIPE_TVALVE					18
 #define PIPE_MANIFOLD4W				19
 #define PIPE_CAP					20
-#define PIPE_OMNI_MIXER				21
-#define PIPE_OMNI_FILTER			22
 #define PIPE_UNIVERSAL				23
```
Comment (AffectedArc07) [inline code/__DEFINES/pipes.dm:24]: My reason for not updating the numbers further down (changing 23 to 21 and vice versa) is I am worried that a lot of atmos devices will break, and I want this to not break things.

**Review by Kyep — APPROVED** [review]
I tried their UI. It is horrible.
I searched myself - they seem to be completely unused.
They're just bloating our code, and our nanoui conversion list.
Delete.

**Comment by TDSSS** [comment]
What is this device even supposed to do?

File: code/__DEFINES/pipes.dm (hunk 1 above)
Comment (TDSSS) [reply code/__DEFINES/pipes.dm:24]: Would it maybe make sense to put a comment or something here explaining this?

**Comment by AffectedArc07** [comment]
> What is this device even supposed to do?

@TDSSS the omni mixer allows you to take 3 mix inputs in any direction you want
```
   [I]
[I][X][I]
   [O]
```
(3 **I**nputs, 1 **O**utput)

The omni filter allows you to filter off 2 gasses instead of just 1
```
   [I]
[1][X][2]
   [3]
```
I = input
1 = Nitrogen out
2 = Oxygen out
3 = Everything else out

**Comment by TDSSS** [comment]
Hmm, I mean if we _are_ in the process of giving everything a new UI then the old UI being shit shouldn't matter. The question is how hard they are to get working, I suppose.

**Comment by AffectedArc07** [comment]
> Hmm, I mean if we _are_ in the process of giving everything a new UI then the old UI being shit shouldn't matter. The question is how hard they are to get working, I suppose.

@TDSSS If you want to make them work, you are more than welcome to try to. For now, considering these are so awful, not used anywhere, and didn't work for me, they may as well be axed.

**Review by variableundefined — APPROVED** [review]
That's a whole lotta code for nothing

**Pull request merged**